src/utils/hints/tokensHints.js

```diff
@@ -0,0 +1,5 @@
+export const tokenHints = {
+  tokenSymbol: 'Symbol of a token. Symbol does not have to be unique.',
+  tokenName: 'Full name of a token.',
+  tokenContractId: 'Identifier of a Smart Contract that implements AEX-9 interface.',
```

Comment:
the smart contract? its shown in the specific location of a single identifier right?

Reply:
Yes, here make sense. Fixed